### ISSUE TITLE
Log collector

### ISSUE THREAD

**Issue by heroinedor** [tchapgouv/synapse-on-k8s#222]
Install a log collector to track the past accesses / actions.

**Comment by ad2ien** [comment]
Yes, I guess we'll be using [loki](https://grafana.com/docs/loki/latest/setup/install/helm/)

**Comment by heroinedor** [comment]
Please [add your planning poker estimate](https://app.zenhub.com/workspaces/EIMIS-6419ddbd85693a00127de149/issues/eimis-ans/eimis-synapse/222?planning-poker) with Zenhub @ad2ien
